packages/@aws-cdk/aws-ec2/lib/vpc-endpoint.ts

```diff
@@ -113,7 +113,10 @@ export interface GatewayVpcEndpointOptions {
   /**
    * Where to add endpoint routing.
    *
-   * @default private subnets
+   * By default, this endpoint will be routable from all subnets in the VPC.
+   * Specify a list of subnet selection objects here to be more specific.
+   *
```

Comment:
Maybe provide an inline "@example". The name `subnets` has been confusing people in untyped languages like Python since the dawn of time.